### DISCUSSION TITLE
Access kbbx on Google Drive

### DISCUSSION THREAD

**Discussion by joaonivaldo** [keepassxreboot/keepassxc#11019] [Q&A] [answered]
Hi, I would like to know if the Windows and Linux client has any option to open the kdbx file directly on Google Drive without having the client installed on the system. I ask this because I don't like leaving the Google client installed on my PC. Thanks

**Comment by phoerious** [comment, marked as answer]
No, you have to use the desktop sync client.

**Reply by joaonivaldo** [reply]
Is there any plan for the implementation of this feature, please?

**Reply by phoerious** [reply]
At the moment, no.

**Reply by michaelk83** [reply]
There is #7222 , but that depends what connectivity Google Drive allows.
